frontend/system/src/queryKeys.ts

```diff
@@ -2,3 +2,4 @@ export const ENTITY_KEY = 'ENTITY';
 export const ENTITIES_KEY = 'ENTITIES';
 export const PENDING_CHANGE_KEY = 'PENDING_CHANGE';
 export const PENDING_CHANGES_KEY = 'PENDING_CHANGES';
+export const PROPERTY_CONFIG_ITEMS_KEY = 'PROPERTY_CONFIG_ITEMS_KEY';
```

Comment:
Very small thing but usually don't have `_KEY` on the end here...
```suggestion
export const PROPERTY_CONFIG_ITEMS_KEY = 'PROPERTY_CONFIG_ITEMS';
```